packages/admin-ui/src/switch/switch.style.ts

```diff
@@ -0,0 +1,127 @@
+import { colors, focusVisible } from '@vtex/admin-ui-core'
+import type { StyleProp } from '@vtex/admin-ui-core'
+import { get } from '@vtex/admin-ui-util'
+
+const css = (csx: StyleProp) => csx
+
+const green40 = get(colors, 'green40')
+const green50 = get(colors, 'green50')
+const gray40 = get(colors, 'gray40')
+
+// State styles
+const active = css({ bg: green40 })
+const inactive = css({
+  height: '1.25rem',
+  width: '2.25rem',
+  top: '0.06rem',
+  left: '0.06rem',
+  marginRight: '0.125rem',
+})
+
+const hoverInactive = css({ bg: gray40 })
+const hoverActive = css({
+  bg: green50,
+})
+
+const trackDefault = css({
+  bg: '$form.neutralInactive',
+})
+
+const focusActive = css({ '&:hover': hoverActive })
+const focusInactive = css({ '&:hover': hoverInactive })
```

Comment:
IMO these consts are not necessary, you can use the values directly instead. It's better for the readability

Reply:
We repeat those at least twice on this style sheet. It occurs to me that this might be better than writing them individually where it's necessary, for maintainability.